content/bn/docs/reference/glossary/admission-controller.md

```diff
@@ -0,0 +1,22 @@
+---
+title: এডমিশন কন্ট্রোলার
+id: admission-controller
+date: 2019-06-28
+full_link: /bn/docs/reference/access-authn-authz/admission-controllers/
+short_description: >
+  কোডের একটি অংশ যা অবজেক্টের অধ্যবসায়ের পূর্বে কুবারনেটিস এপিআই(API) সার্ভারের অনুরোধগুলিকে বাধা দেয়।
```

Comment:
@cloudwithtanvir, could you kindly review this line?

Reply:
@asem-hamid @sajjadrahman56 
কোডের একটি অংশ যা অবজেক্টের পার্সিস্টেন্সের পূর্বে কুবারনেটিস এপিআই(API) সার্ভারের অনুরোধগুলিকে বাধা দেয়।

Note: পার্সিস্টেন্সের সহজ বাংলা হলো স্থায়ীভাবে সংরক্ষিত রাখা , However, I prefer to keep the term "persistence" in English, as it is widely understood in the technology field.

Reply:
@cloudwithtanvir  Thank you 

@sajjadrahman56 replace with this 

কোডের একটি অংশ যা অবজেক্টের পার্সিস্টেন্সের(persistence) পূর্বে কুবারনেটিস API সার্ভারের অনুরোধগুলিকে বাধা দেয়।